.github/workflows/prerelease.yml

```diff
@@ -6,40 +6,46 @@ on:
       - next
 
 jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: hallee/eslint-action@1.0.3
```

Comment:
Seems like this is abandoned:
https://github.com/hallee/eslint-action#this-repository-has-been-abandoned

Reply:
🤔 hmm neat, so I just run eslint in my workflow anyways and it surfaces the info?

Reply:
Yeah!